### ISSUE TITLE
@aspnet/signalr-client syntax error on IE

### ISSUE THREAD

**Issue by jacopozzo** [aspnet/SignalR#1363]
Hi everybody,
I've had no problem running @aspnet/signalr-client on Edge/Chrome or other browser, but using IE 11 (which does not support ES6) it has raised a syntax error as following:
![image](https://user-images.githubusercontent.com/19947058/35516252-93379d66-050b-11e8-8f55-65bdcb7022f5.png)
![image](https://user-images.githubusercontent.com/19947058/35519323-3184e002-0514-11e8-8dc3-7f90202b4524.png)

Using the ES5 compatible library ( signalr-clientES5-1.0.0-alpha2-final.js ), i've no errors, but i noticed a weird behaviour trying to broadcasting a simple message to clients.

When i press "Send", everything run, but clients can see the new messages after, for example, getting and losing focus from the text input.
In other words, it's not immediate like when i use the first one library.

Is a very weird situation and i can't reproduce it as well.

**Comment by anurse** [comment]
In early preview builds (including Alpha 2), the main SignalR library required a browser with native support for ES6 Promises. The ES5 version was designed for those older browsers. However, we've decided in the upcoming Preview 1 (for which nightly builds are available) to shift the default to the ES5 version so that it works in all browsers.

See the [Packages](https://github.com/aspnet/SignalR#packages) and [Deploying](https://github.com/aspnet/SignalR#deploying) sections of the readme for this repo for information on how to use the nightly builds.

Also, please note that the package has been **renamed** in the recent nightly builds. It is now called `@aspnet/signalr`. The UMD bundle for browsers, referred to in the README, is now ES5-compatible and should work in IE.

Going to close this now as that should resolve this issue. Please feel free to re-open or post additional questions/comments if the issue is not yet resolved.

**Comment by jacopozzo** [comment]
Thank you very much.
Now, installing @aspnet/signalr from your registry, it cannot call negotiation correctly and i have the following message:
![image](https://user-images.githubusercontent.com/19947058/35558852-62978bf0-05a9-11e8-9629-3f78740f73eb.png)

my Startup.cs:

` public class Startup
    {
        public IConfigurationRoot Configuration { get; }

        public Startup(IHostingEnvironment env)
        {
            var builder = new ConfigurationBuilder()
                .SetBasePath(env.ContentRootPath)
                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
                .AddEnvironmentVariables();


            Configuration = builder.Build();
        }

        // This method gets called by the runtime. Use this method to add services to the container.
        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
        public void ConfigureServices(IServiceCollection services)
        {
            services
                         .AddMvc()
                         .AddJsonOptions(options => options.SerializerSettings.ContractResolver =
                             new DefaultContractResolver());

            services.AddSignalR();
        }

        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
        {
            loggerFactory.AddConsole(Configuration.GetSection("Logging"));
            loggerFactory.AddDebug();

            app.UseCors(
                builder => builder.AllowAnyOrigin()
                    .AllowAnyHeader()
                    .AllowAnyMethod()
                    .AllowCredentials())
                    .UseDefaultFiles()
                .UseStaticFiles();
            #region old
            //app.UseMvc(routes =>
            //{
            //    routes.MapRoute(
            //        name: "default",
            //        template: "{controller=Home}/{action=Index}/{id?}");
            //});
            #endregion

            app.Use(async (context, next) =>
        {
            await next();

            if (context.Response.StatusCode == 404
                && !Path.HasExtension(context.Request.Path.Value))
            {
                context.Request.Path = "/index.html";
                await next();
            }
        });

            app.UseMvc();

            app.UseSignalR(cfg =>
            {
                cfg.MapHub<TestHub>("chat");
            });
        }`

my component.ts
`ngOnInit() {
        this.nick = window.prompt("What's your nickname:", "J4C0X92");

        this.hubConnection = new HubConnection("/chat");


        this.hubConnection.on('sendToAll', (nick: string, receivedMessage: string) => {
            const text = `${nick}: ${receivedMessage}`;
            this.messages.push(text);
        });
        
        
        this.hubConnection
            .start()
            .then(() => console.log('Connection started!'))
            .catch(err => console.log('Error while establishing connection :(',err));
        
    }`
Seems to be everything ok, but i don't know why client trying to call negotation failing.
